### DISCUSSION TITLE
headless tabs not actually joining the game?

### DISCUSSION THREAD

**Discussion by pmLite** [bprussell/VstheWorld#7] [Q&A]
hello,

I've been attempting to use this tool, but I'm having an issue that the tabs get created and enter the usernames from 1 up to 334, but they never actually appear in the audience of the lobby. I've made sure to start the game with one player and paused, and I've also tried setting the max players to 2 then filling these spots up on my phone before running the script. The "entered room code" and "entered username xxx" are both returned in command prompt, but the audience still shows 0. Please help!

**Comment by bprussell** [comment]
I'm a little bit confused. When you say you set the max players to 2, do you mean you ran the following command?

`npm start ABCD -- --sessions 2`

Can you please paste the console output?

**Reply by pmLite** [reply]
Sorry - I mean I tried changing the maximum number of players in the __Jackbox__ settings to 2, so that any new players would be forced into the audience after that (I had already tried starting the game with one player and then adding the audience with no luck).

I can try running it again tomorrow, but in short, the console returned all the normal messages up to "entered room code" and "entered username xxx" for each of the 334 tabs it was supposed to, BUT they never actually appeared in the game.

I am not at all well-versed in what I'm actually doing here so I was just trying to follow your instructions as closely as possible, but I have a slight theory that it may actually be a network issue. In my numerous attempts earlier today, there would be times when Jackbox would be unable to create a game due to connection issues, or my phone would say there was "no game found" when I entered the room code, despite me still having a normal working Internet connection on both devices - perhaps there isn't enough bandwidth for these tabs or something? I don't know; again, I'm really out of my area of expertise here.

**Reply by bprussell** [reply]
I have a couple of ideas here. First off, make sure you're running the game in Streamer Mode, which allows unlimited audience members. Then you start the game with you and maybe one other player, then pause. That's when you run the script to join as audience. Any people that join the game after it's started counts as an audience member.

You can test whether it's a network issue by trying with just a few tabs. For example, run with `npm start ABCD -- --sessions 10` and that should only try to add 10 players to the game. If you can run that and check on the game, there should be 10 audience members at that point, and then you can start ramping it up to see when your computer can't handle it anymore. 

I had to recruit a friend to help as my computer was not capable of maintaining 1000 connections.
